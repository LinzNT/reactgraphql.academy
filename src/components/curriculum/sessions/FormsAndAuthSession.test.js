import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import FormsAndAuthSession from './FormsAndAuthSession'

describe('<FormsAndAuthSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<FormsAndAuthSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
