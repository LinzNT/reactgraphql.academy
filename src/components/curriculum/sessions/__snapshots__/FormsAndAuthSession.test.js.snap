// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormsAndAuthSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Forms management in React
      <styled.ul>
        <styled.li>
          Controlled Components
        </styled.li>
        <styled.li>
          Uncontrolled Components
        </styled.li>
      </styled.ul>
    </styled.li>
    <styled.li>
      Authentication
      <styled.ul>
        <styled.li>
          JWT
        </styled.li>
        <styled.li>
          Authorization, public and private pages
        </styled.li>
      </styled.ul>
    </styled.li>
  </styled.ul>
</Session>
`;
